src/stories/Library/nav-spot/NavSpot.tsx

```diff
@@ -1,21 +1,36 @@
 import { FC, ReactNode } from "react";
+import clsx from "clsx";
 import { ReactComponent as Arrow } from "../Arrows/icon-arrow-ui/icon-arrow-ui-large-right.svg";
 
 type NavSpotProps = {
   variant?: string;
   title: string;
   subtitle?: string;
   media?: ReactNode;
+  placeholderText?: string;
 };
 
-const NavSpot: FC<NavSpotProps> = ({ variant, title, subtitle, media }) => {
+const NavSpot: FC<NavSpotProps> = ({
+  variant,
+  title,
+  subtitle,
+  media,
+  placeholderText,
+}) => {
+  const classes = clsx("nav-spot", "arrow__hover--right-large", {
+    "nav-spot--has-no-media": !media,
+    "nav-spot--has-media": media,
+  });
+
   return (
-    <article
-      className="nav-spot arrow__hover--right-large"
-      data-variant={variant}
-    >
+    <article className={classes} data-variant={variant}>
       <a href="#" className="nav-spot__content">
-        {media ? <figure className="nav-spot__media">{media}</figure> : ""}
+        <figure className="nav-spot__media">
+          {media || (
+            <div className="nav-spot__placeholder-text">{placeholderText}</div>
+          )}
+        </figure>
+
         <div className="nav-spot__text">
           <h1 className="nav-spot__title">{title}</h1>
```

Comment:
It is outside the scope of this PR and heading levels are tricky but I would never expect a `<h1>` to be the right heading level for a Nav Spot.

Reply:
It's wrapped in a `<article>`, so yeah, h1 is appropiate :) @kasperg